README.md

```diff
@@ -1,107 +1,31 @@
 >ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.
 
-# SignalFx Distributed Tracing Instrumentation
+# Tracing examples for Splunk Observability Cloud
 
-## Instrumentation Agnostic
+This repository contains tracing examples for Splunk Observability Cloud, formerly
+known as SignalFx. Most of the examples use Splunk distributions of OpenTelemetry
+instrumentations. To learn more about OpenTelemetry and Splunk Observability Cloud,
+see [the official documentation](https://docs.splunk.com/Observability).
 
-SignalFx Distributed Tracing intends to be instrumentation agnostic; supporting
-& ingesting variety of popular open instrumentation libraries including
-[OpenTracing](http://opentracing.io/), [Zipkin](https://zipkin.io/) &
-[OpenCensus](https://opencensus.io/). So long as the tracer configured to send
-spans to SignalFx ingest endpoint uses the Zipkin v1/2 JSON wire format or
-Jaeger Thrift format, we will accept spans irrespective of how they were
-instrumented - via one of the above mentioned open libraries or a homegrown one.
-Our goal is to build on the shoulders of giant communities rather than re-invent
-the wheel with proprietary libraries/agents and give customers choice to decide
-what works for them without worrying about vendor lock-in.
+## Supported formats
 
-<p align="center">
-  <a href="https://opentracing.io">
-  <img src="https://avatars2.githubusercontent.com/u/15482765?s=100&v=4"
-       alt="OpenTracing" /></a>
-  &nbsp;&nbsp;
-  <a href="https://jaegertracing.io">
-  <img src="https://avatars3.githubusercontent.com/u/28545596?s=100&v=4"
-       alt="Jaeger Tracing" /></a>
-  &nbsp;&nbsp;
-  <a href="https://zipkin.io">
-  <img src="https://avatars3.githubusercontent.com/u/11860887?s=100&v=4"
-       alt="ZipKin" /></a>
-    &nbsp;&nbsp;
-  <a href="https://opencensus.io">
-  <img src="https://avatars3.githubusercontent.com/u/26944525?s=100&v=4"
-       alt="OpenCensus" /></a>
-</p>
+Splunk Observability Cloud and the Splunk Distribution of OpenTelemetry Collector
+can ingest spans in the following formats:
```

Comment:
```suggestion
Splunk Observability Cloud and the Splunk Distribution of OpenTelemetry Collector
can ingest tracing data in the following formats:
```

The list is not comprehensive but the statement reads like it is. Is there a way to say "these are the relevant/interesting formats we cover in this repo" or something?

Reply:
Sure! Adding